src/components/CrossSell/CrossSell.jsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import {
+  CrossSellStyled,
+  CrossSellHeader,
+  CardsContainer,
+  ButtonContainer,
+  Footer,
+  Card,
+  CustomH3,
+  ButtonMargin,
+  SVGContainer,
+} from './style';
+import Text from '../Text/Text';
+import Button from '../Button/Button';
+
+export default class CrossSell extends React.Component {
+  renderHeader = () => {
+    const { texts } = this.props;
+
+    return (
+      <CrossSellHeader>
+        <Text type="h1">{texts.title}</Text>
+        <Text type="h3">{texts.subtitle}</Text>
+      </CrossSellHeader>
+    );
+  };
+
+  renderBody = () => {
+    const { texts } = this.props;
+
+    return (
+      <CardsContainer>
+        {texts.cards.map(card => (
+          <Card>
+            <img src={card.image} alt="kitty" />
+            <CustomH3>{card.title}</CustomH3>
```

Comment:
Also, curious what is this magical `CustomH3` component 🤔 ?

Reply:
hahahahahaha oh noooo! Great catch @ivanaszuber , thanks for pointing it out!